front/app/components/UI/LeafletMap/simplestyle.ts

```diff
@@ -142,3 +142,5 @@
 
   L.GeoJSON.addInitHook('_useSimpleStyle');
 });
+
+export {};
```

Comment:
What's the purpose of this?

Reply:
In order to make `esbuild` work correctly, we needed to add `"isolatedModules": true` to our `tsconfig`. The reason for the import/export changes is that with this setting on:
1. Whenever you re-export a type, you need to make that explicit with `export type`
2. Everything should be a module so whenever a file is not, we add `export {};` in order to convert it to one.